### PR TITLE
Delete empty trash dirs

### PR DESCRIPTION
**Pull request by dragonchaser**
This PR adds a cli command that allows purging empty dirs in trash folder structures.

refs #9271 
refs #9393

### CODE REVIEW

**Review by 2403905 — APPROVED** [review]

File: changelog/unreleased/empty-trash-dirs.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Empty trash directories
+
+We have added a cli-command that allows cleaning up emppty directories in the trashbins folder structure in decomposedFS.
```
Comment (kobergj) [inline changelog/unreleased/empty-trash-dirs.md:3]:
```suggestion
We have added a cli-command that allows cleaning up empty directories in the trashbins folder structure in decomposedFS.
```

File: ocis/pkg/command/trash.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+		Action: func(_ *cli.Context) error {
+			fmt.Println("Read the docs")
+			return nil
+		},
+	}
+}
+
+func TrashPurgeOrphanedDirsCommand(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:  "purge-orphaned-dirs",
```
Comment (kobergj) [inline ocis/pkg/command/trash.go:33]:
```suggestion
		Name:  "purge-empty-dirs",
```
They are not orphaned, they are empty. That is a difference. Please bulk change.

File: ocis/pkg/command/trash.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+				Name:  "dry-run",
+				Usage: "do not delete anything, just print what would be deleted",
+				Value: true,
+			},
+		},
+		Action: func(c *cli.Context) error {
+			basePath := c.String("basepath")
+			if basePath == "" {
+				fmt.Println("basepath is required")
+				return cli.ShowCommandHelp(c, "consistency")
```
Comment (kobergj) [inline ocis/pkg/command/trash.go:52]:
```suggestion
				return cli.ShowCommandHelp(c, "trash")
```

**Review by kobergj — CHANGES_REQUESTED** [review]

File: ocis/pkg/command/trash.go [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+			fmt.Println("Read the docs")
+			return nil
+		},
+	}
+}
+
+func TrashPurgeOrphanedDirsCommand(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:  "purge-empty-dirs",
+		Usage: "purge orphaned directories",
```
Comment (kobergj) [inline ocis/pkg/command/trash.go:34]: please bulk change `orphaned` -> `empty` everywhere in this PR. grepping for orphaned should not yield any results.

**Review by kobergj — CHANGES_REQUESTED** [review]

**Review by kobergj — APPROVED** [review]
Nice and clean :+1:

**Pull request merged**